CHANGES.md

```diff
@@ -93,6 +93,13 @@ next
 
 - Make `dev` the default build profile (#920, @diml)
 
+- Add the ability to build an alias non-recursively from the command
+  line by writing `@@alias` (#926, @diml)
```

Comment:
About the syntax: I wonder if instead (or in addition to) this `@@alias` syntax, we could use a directory-based syntax like `@./alias`. Maybe that could be extended to other directories to support #771?

Reply:
You can already write a directory after `@` (and `@@` in this PR). For instance `@foo/runtest` means alias `runtest` recursively from `foo` and `@@foo/runtest` means alias runtest only in `foo`.